### PR TITLE
Added font size setting and dark mode

### PR DESCRIPTION
**Pull request by filippocld**
Added a simple font size chooser. Sorry but the default size looked ridiculous to me.

### CODE REVIEW

**Comment by filippocld** [comment]
Sorry for the double purpose commit. I should have done 2 separate pull requests :-( 
It includes both font size and dark mode.
Is it a problem?

**Comment by louisdh** [comment]
That's ok

File: Textor/Controller/SettingsViewController.swift [hunk 1]
```diff
@@ -48,11 +55,22 @@ class SettingsViewController: UITableViewController {
...
-	
+    
+    @IBAction func fontSizeChanged(_ sender: UIStepper) {
+        UserDefaults.standard.set(sender.value, forKey: "fontSize")
+        fontSizeLabel.text = "\(Int(sender.value))"
+    }
+    
+    @IBAction func themeChanged(_ sender: UISwitch) {
+        UserDefaults.standard.set(sender.isOn, forKey: "darkMode")
+        let alert = UIAlertController(title: "Theme updated!", message: "Restart the app for changes to take effect", preferredStyle: .alert)
```
Comment (louisdh) [inline Textor/Controller/SettingsViewController.swift]: I'm not really a fan of "Restart the app for changes to take effect". Ideally the app would change the theme without restart.

**Comment by filippocld** [comment]
Added the ability to change theme without restarting 👍

**Comment by louisdh** [comment]
Thanks!

**Pull request merged**